libnopegl/src/node_textureview.c

```diff
@@ -59,7 +59,7 @@ static int textureview_init(struct ngl_node *node)
     }
 
     if (o->texture->cls->id == NGL_NODE_TEXTURECUBE && o->layer >= 6) {
-        LOG(ERROR, "cubemap textures only have 6 layers");
+        LOG(ERROR, "requested layer (%d) exceeds cube map layer count (%d) cubemap textures only have 6 layers", o->layer, texture_params->depth);
```

Comment:
weird sentence break between "layer count" and "cubemap textures" 🤔

Maybe a comma or something would help?

I'm also seeing 3 variables: the requested layer, the cubemap layer count, and "6". Is `texture_params->depth` always 6 here?

Reply:
Fixed.

Reply:
Did you push?

Reply:
I did not, sorry. Fixed.